src/Scrambler.test.js

```diff
@@ -45,7 +45,16 @@ describe('Scrambler', () => {
     const defaultCharacters = Scrambler.CHARACTERS.DEFAULT;
     defaultCharacters.includes('*', '@');
 
+    const digitCharacters = Scrambler.CHARACTERS.DIGITS;
+    digitCharacters.includes('0', '1', '2', '3', '4');
+
     const alphabetCharacters = Scrambler.CHARACTERS.ALPHABET;
     alphabetCharacters.includes('a', 'b', 'c', 'd', 'e');
+
+    const alphabetUpperCaseCharacters = Scrambler.CHARACTERS.ALPHABET_WITH_CAPITALS;
+    alphabetUpperCaseCharacters.includes('A', 'B', 'C', 'a', 'b', 'c');
+
+    const combinedCharacters = Scrambler.CHARACTERS.DIGITS_AND_ALPHABET_WITH_CAPITALS;
+    combinedCharacters.includes('0', '1', '2', 'A', 'B', 'C', 'a', 'b', 'c');
```

Comment:
It would be great if there are any use cases for the added parameters(`maxCounter`, `delay`) in the test cases.

Reply:
I agree. Added the new parameters after these changes, so they are not included here. Happy to make the necessary additions. Let me know how you'd prefer to proceed.

Reply:
There is currently a test case for `"with the option to set the characters to use when scrambled."` We could extend it to something like this:

```js
describe('with custom characters, speed, and shuffleCount', () => {
    it('scrambles text according to the provided options', () => {
      const scrambler = new Scrambler();

      texts.forEach((text) => {
        scrambler.scramble(text, handleScramble, {
          characters: ['a', 'b', 'c'],
          speed: 2,          
          shuffleCount: 20,
        });
        expect(handleScramble).toHaveBeenLastCalledWith(text);
      });
    });
  });
```

This way, we can demonstrate the use cases to the users through the test cases.

Also, it would be good to add test cases for input validation of each parameter. If you encounter any difficulties, please feel free to let me know.